src/index.js

```diff
@@ -11,8 +11,12 @@ const createGroups = (pages, pageLength) =>
 
 const getPageIndex = index => (index === 0 ? `` : index + 1)
 
-const buildPaginationRoute = (index, pathPrefix) =>
-  index > 1 ? `${pathPrefix}/${index}` : pathPrefix
+const buildPaginationRoute = (index, pathPrefix) => {
+  if (!index && !path) {
```

Comment:
`path` should actually be `pathPrefix`, no?

Reply:
just noticed, you are right!